src/index.html

```diff
@@ -7,7 +7,173 @@
+            Over 50 years experience in love.
+          </h3>
+          <p class="about-article__text-2">
+            Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical. <br><br>The market leader in real estate since 2003. Awards Real Estate Company, 2010, 2011, 2013. More than 200 satisfied customers premium segment. We focus only on new buildings and high-level thoroughly know this area better than other specialists. Working directly with developers - so have more useful information about the profitable deals. <br><br> <strong>We will never be late! <br> Guaranteed secure transaction - 100%</strong>
+          </p>
+        </article>
+      </section>
+
+      <section class="gallery" id="_gallery">
+        <h2 class="gallery__header">Gallery</h2>
+        <div class="gallery__photos">
+          <div class="gallery__section">
+            <img src="images/photos/1.jpg" alt="marina palms" class="gallery__photo_1">
+            <img src="images/photos/2.jpg" alt="marina palms" class="gallery__photo_2">
+          </div>
+          <p class="gallery__info">
+            <span class="gallery__bluish_txt">Marina Palms</span> / North Miami Beach, FL 33162
+          </p>
+          <div class="gallery__section">
+            <img src="images/photos/3.jpg" alt="edition residens" class="gallery__photo_3">
+            <img src="images/photos/4.jpg" alt="edition residens" class="gallery__photo_4">
+          </div>
+          <p class="gallery__info">
+            <span class="gallery__bluish_txt">Edition Residens</span> / Miami Beach, FL 33139
+          </p>
+          <div class="gallery__section">
+            <img src="images/photos/5.jpg" alt="faena house" class="gallery__photo_5">
+            <img src="images/photos/6.jpg" alt="faena house" class="gallery__photo_6">
+          </div>
+          <p class="gallery__info">
+            <span class="gallery__bluish_txt">Faena House</span> / Miami Beach, FL 33140
+          </p>
+          <button class="gallery__button">View all</button>
+        </div>
+      </section>
+
+      <section class="consultation" id="_consultation">
+        <h2 class="consultation__header">Consultation</h2>
+        <div class="consultation__block">
+          <article class="consultation-article">
+            <h3 class="consultation-article__header">
+              How long have you do yourself a gift?
+            </h3>
+            <p class="consultation-article__par">
+              The procedural change mezzo forte starts rhythmically pickup at these moments stop LA Mazel and VA Tsukkerman in his "Analysis of musical works." Pointillism, which originated in the music of the early twentieth century, microforms, found a distant historical.
+            </p>
+          </article>
+
+          <form action="#" method="post" class="consultation__form">
```

Comment:
метод пишется в верхнем регистре
```suggestion
          <form action="#" method="GET" class="consultation__form">
```
```suggestion
          <form action="#" method="POST" class="consultation__form">
```